manifest.json

```diff
@@ -0,0 +1 @@
+{"icons":[],"display":"fullscreen"}
```

Comment:
What is this file for?  Do we need it?

Reply:
We technically don't need it for our purpose but it's always good to have one 
more about it here -> https://developers.google.com/web/fundamentals/web-app-manifest

Reply:
Thank you --- I was not familiar with this standard!  I agree with your including it.